core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -136,4 +146,22 @@ class SingleAppSummaryInfoProvider(val app: ApplicationSummaryInfo)
   override def getRapidsJars: Seq[String] = {
     app.rapidsJar.map(_.jar).seq
   }
+
+  override def hasFileCacheSupportedFormat: Boolean = {
+    app.dsInfo.exists { info =>
+      info.format.contains("Parquet")
+    }
+  }
+
+  override def getDistinctLocationPct: Double = {
+    100.0 * app.dsInfo.groupBy(_.location).size / app.dsInfo.size
+  }
+
+  override def getRedundantReadSize: Long = {
+    app.dsInfo.groupBy(_.location)
+      .filter { case(_, objects) => objects.size > 1 }
```

Comment:
Note that we would want to further qualify the redundant reads with ones that are supported by filecache.  Arguably we could just fold the format filter into `getRedundantReadSize` so it will return 0 for any redundant reads to file formats not supported by filecache.  Then we don't need to check separately if any reads are supported by filecache, since we only care about redundant reads supported by filecache.

Reply:
Good catch.  Updated in latest commit.